lib/screens/home/home.dart

```diff
@@ -67,9 +68,66 @@ class _HomeState extends State<Home> with TickerProviderStateMixin {
     }
   }
 
+  void loadPreferences() async {
+    SharedPreferences prefs = await SharedPreferences.getInstance();
+    String sort = prefs.get('FeedSortSelection');
+    switch (sort) {
+      case 'OverallScore':
+        setState(() {
+          sortType = SortType.overAllScore;
+          titleFilter = "Top";
+        });
+        break;
+      case 'RecentlyUpdated':
+        setState(() {
+          sortType = SortType.recentlyUpdated;
+          titleFilter = 'Updated';
+        });
+        break;
+      case 'NewestPackage':
+        setState(() {
+          sortType = SortType.newestPackage;
+          titleFilter = 'New';
+        });
+        break;
+      case 'Popularity':
+        setState(() {
+          sortType = SortType.popularity;
+          titleFilter = 'Popular';
+        });
+        break;
+      default:
+        break;
+    }
+    String filter = prefs.get('FeedFilterSelection');
```

Comment:
This could be shrunk into a single setState call with the variable assignment inside once you figure out which variable, or you can put the switch statement inside it